stories/test_data/bar_chart_with_2_metrics_and_2_view_by_attributes_request.json

```diff
@@ -0,0 +1,82 @@
+{
+    "execution": {
+        "afm": {
+            "measures": [
+                {
+                    "localIdentifier": "e70588a477ce487fbd01ff001c1848b0",
+                    "definition": {
+                        "measure": { "item": { "uri": "/gdc/md/dfnkvzqa683mz1c29ijdkydrsodm8wjw/obj/1279" } }
```

Comment:
Please replace all real `dfnkvzqa683mz1c29ijdkydrsodm8wjw ` project id to fake ones, like `projectId`
And also consider for `localIdentifier` value of both measures and attributes, like `amountIdentifier`, `productIdentifier` etc

Reply:
I will move these into test-cafe in AD